core/src/test/scala/unit/kafka/server/SimpleFetchTest.scala

```diff
@@ -69,8 +69,10 @@ class SimpleFetchTest {
   @Before
   def setUp() {
     // create nice mock since we don't particularly care about zkclient calls
-    val zkUtils = EasyMock.createNiceMock(classOf[ZkUtils])
-    EasyMock.replay(zkUtils)
+    val zkClient = EasyMock.createNiceMock(classOf[ZkClient])
```

Comment:
I agree that these changes (and also in the other file) are weird to include here.